### PR TITLE
Issue16 remove nova

### PR DESCRIPTION
**Pull request by dtregonning**
Removal of Nova references and adding of Apache licensing file and headers

### CODE REVIEW

**Comment by luckyj5** [comment]
Need to remove docker-logging-plugin binary file. 
Rest looks good. Thanks @dontregonning

File: message_processor.go [hunk 1]
```diff
@@ -64,6 +80,7 @@ func (mg messageProcessor) consumeLog(lf *logPair) {
 }
 
 // send the log entry message to logger
+
```
Comment (sharonx) [inline message_processor.go]: remove this empty line

File: message_processor.go [hunk 2]
```diff
@@ -100,4 +117,4 @@ func (mg messageProcessor) shouldSendMessage(message []byte) bool {
 		return true
 	}
 	return true
-}
+}
```
Comment (sharonx) [inline message_processor.go]: Revert the change here: goLint requires an empty line at the end of each file.

**Review by sharonx — COMMENTED** [review]
Please make the two minor changes mentioned and remove docker-logging-plugin binary per @luckyj5

**Comment by dtregonning** [comment]
@sharonx @luckyj5 - Thanks guys. Fixed

**Review by sharonx — APPROVED** [review]
